superset/assets/src/featureFlags.js

```diff
@@ -0,0 +1,11 @@
+// A higher-order function that takes the redux state tree and returns a
+// `isFeatureEnabled` function which takes a feature and returns whether it is enabled.
+// Note that we assume the featureFlags subtree is at the root of the redux state tree.
+export function isFeatureEnabledCreator(state) {
+  return feature => !!state.featureFlags[feature];
+}
+
+// Feature flags are not altered throughout the life time of the app
+export default function featureFlagsReducer(state = {}) {
```

Comment:
do you think this is necessary if it's never modified?

Reply:
I think it feels odd to have a subtree of the redux state tree dangling without a reducer. This also leaves room for enriching the feature flag system in the future if we'd like (e.g. allowing toggling of a feature flag without restarting the server). Thoughts?

Reply:
Feature flag override would be interesting. Some apps has Chrome plugin that you can turn on/off experimental features. With that said I am OK with having empty reducer here for now.